### PR TITLE
Reworks the DLL-Plugin so that its possible to explicitly include /

### PR DESCRIPTION
**Pull request by strues**
…exclude dependencies.

**Note:** I've included lodash for ease of use and speed since this was a PoC. If the consensus is to get rid of lodash, we'll need standard functions for pullAll, uniq and defaultDeep.

### CODE REVIEW

**Comment by ctrlplusb** [comment]
Hi @strues!

Thanks for making a start on this.  I'll start reviewing and we can work from there.

Off the bat though I would prefer if we dropped the lodash dep.  I'm a ramda man myself, but would prefer the starter be unopinionated in this regard.

File: tools/development/createVendorDLL.js [hunk 1]
```diff
@@ -47,66 +50,25 @@ function createVendorDLL(bundleName : string, bundleConfig : Object) {
     };
   }
 
-  function extractModulesFromSrcFiles(fileCollections) {
```
Comment (ctrlplusb) [inline tools/development/createVendorDLL.js]: NICE!

**Comment by strues** [comment]
Lodash is gone. Check things out. Everything seems to work, but some thorough testing wont hurt.

**Comment by ctrlplusb** [comment]
Awesomeness. I will give it a bash as soon as this config story settles.  Do you mind adding me as being allowed to commit on the fork?

**Comment by strues** [comment]
@ctrlplusb its actually a branch in this repo because I didnt realize I wasnt in my fork.

**Comment by ctrlplusb** [comment]
Brilliant, I am going to wait for `next` to settle down a bit and then mere it into there. :)

**Comment by ctrlplusb** [comment]
Thanks @strues! ❤️

**Pull request merged**